flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/StateAssignmentOperation.java

```diff
@@ -640,4 +639,17 @@ private static void checkStateMappingCompleteness(
 
 		return subtaskKeyedStateHandles;
 	}
+
+	private static <T extends AbstractChannelStateHandle<?>> OperatorStateRepartitioner<T> nonRescalingPartitioner(String logStateName) {
```

Comment:
I think there is no UT or IT to cover this method currently in this PR (e.g. throw expected exception).

Reply:
That's right. I'm adding some new tests including `ChannelStateNoRescalingPartitionerTest`.'